src/config/dynamic-theme-fixes.config

```diff
@@ -3883,6 +3883,8 @@ developers.google.com
 
 INVERT
 .devsite-google-wordmark
+a.devsite-nav-title.devsite-nav-active
+div.devsite-article-meta.nocontent
```

Comment:
These are not relevant to this PR.

Reply:
Yeah, I pushed then from GitHub Desktop forgetting that they would go into the same pull request automatically. 😅 We can delete them or something and I'll do a different pull request for them some other time. I won't be available much today unfortunately.